cid.go

```diff
@@ -51,15 +51,16 @@ var (
 // the codes described in the authoritative document:
 // https://github.com/multiformats/multicodec/blob/master/table.csv
 const (
-	Raw = 0x55
-
-	DagProtobuf = 0x70
-	DagCBOR     = 0x71
-	Libp2pKey   = 0x72
-
-	GitRaw = 0x78
-
-	DagJOSE               = 0x85
+	// core IPLD
+	Raw         = 0x55
```

Comment:
Maybe it's worth deprecating these and pointing to https://github.com/multiformats/go-multicodec even though we're going to leave them in.